cfme/tests/test_login.py

```diff
@@ -5,8 +5,12 @@
 from cfme.configure.access_control import User
 from cfme.utils.appliance import ViaSSUI, ViaUI
 from cfme.utils import conf, error
+from cfme.utils.version import current_version
 
-pytestmark = pytest.mark.usefixtures('browser')
+pytestmark = [
+    pytest.mark.usefixtures('browser'),
+    pytest.mark.uncollectif(lambda: current_version() > '5.8')
```

Comment:
Since this will also uncollect the OPS UI tests, which we want to keep in smoke tests, I think this needs to be done with a skip inside test_login on version 5.9+ and context==ViaSSUI